### DISCUSSION TITLE
1주차 Discussion

### DISCUSSION THREAD

**Discussion by ddongule** [elegant-functional-2023/javascript-sicp-2023#1] [Announcements]
## 📢 책을 읽으면서 궁금한 점들을 자유롭게 적고 대답해주세요!

**Comment by sunhpark42** [comment]
부동소수점 연산
: https://ko.wikipedia.org/wiki/%EB%B6%80%EB%8F%99%EC%86%8C%EC%88%98%EC%A0%90
: https://docs.oracle.com/cd/E19957-01/806-3568/ncg_goldberg.html
